lib/domain/repositories/marvell_repository.dart

```diff
@@ -1,8 +1,10 @@
 import 'package:marvelapp_flutter/domain/entities/character.dart';
 import 'package:marvelapp_flutter/domain/entities/series.dart';
+import 'package:dartz/dartz.dart';
+import 'package:marvelapp_flutter/domain/error_handling/failure.dart';
 
 abstract class MarvellRepository {
-  Future<List<Character>> getCharacters([int offset]);
-  Future<Character> getCharacterDetail(String characterId);
-  Future<List<Series>> getSeries(String characterId);
+  Future<Either<Failure, List<Character>>> getCharacters([int offset, bool getFromLocal]);
```

Comment:
`getFromLocal ` should not be in Repository, domain should not know its local or remote

Reply:
done